src/cdk/layout/layout.md

```diff
@@ -2,7 +2,7 @@ The `layout` package provides utilities to build responsive UIs that react to sc
 
 ### BreakpointObserver
 
-`BreakpointObserver` is a utility for evaluating media queries and reacting to their changing.
+`BreakpointObserver` is a utility for evaluating media queries and reacting to their changes.
```

Comment:
I think the wording here actually needs to be changed in a different way. This sentence implies that the media queries are the thing changing, when it's actually the _result_ of the query. What about something like
```markdown
`BreakpointObserver` is a utility for evaluating media queries and reacting to changes
in the results of those queries.
```

Reply:
Updated